installation/scripts/copy-resource.ps1

```diff
@@ -2,7 +2,7 @@ $CURRENT_DIR = Split-Path $MyInvocation.MyCommand.Path
 $LOCAL_DIR = "${CURRENT_DIR}\..\..".Substring(2)
 $INSTALLER_NS = "kyma-installer"
 $INSTALLER_POD = "kyma-installer"
-$REMOTE_DIR = "/kyma"
+$REMOTE_DIR = "/kyma/injected"

```

Comment:
for me it is not obvious what kyma/injected means, maybe "kyma/installer_input" will be the more meaningful name?